### PR TITLE
(maint) Update docs to include fact caching

### PR DESCRIPTION
**Pull request by Magisus**
This updates the readme and the man page to include information about
fact caching. Also updates the POT file to include all recently added
strings.

### CODE REVIEW

File: locales/FACTER.pot [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # SOME DESCRIPTIVE TITLE.
 # Copyright (C) YEAR Puppet <docs@puppet.com>
-# This file is distributed under the same license as the FACTER package.
+# This file is distributed under the same license as the PACKAGE package.
```
Comment (Magisus) [inline locales/FACTER.pot]: Does anyone know why this happened? Seems wrong.

**Comment by Magisus** [comment]
Since this concerns functionality that was released as part of 3.5.0 (now on stable), targeting this at stable.

File: man/man8/facter.8 [hunk 2]
```diff
@@ -27,28 +27,32 @@ Collect and display facts about the current system\. The library behind Facter i
...
-      \fB\-\-color\fR                      Enables color output\.
-      \fB\-\-config\fR                     A custom location for the chnfig file\.
+      \fB\-\-color\fR                      Enable color output\.
+      \fB\-\-config\fR                     A custom location for the config file\.
       \fB\-\-custom-dir\fR arg             A directory to use for custom facts\.
 \fB\-d, [ \-\-debug ]\fR                    Enable debug output\.
       \fB\-\-external-dir\fR arg           A directory to use for external facts\.
       \fB\-\-help\fR                       Print help and usage information\.
 \fB\-j, [ \-\-json ]\fR                     Output facts in JSON format\.
+      \fB\-\-list\-block\-groups\fR          List the names of all blockable fact groups\.
```
Comment (MikaelSmith) [inline man/man8/facter.8]: Does this all print with the right spacing?

Comment (MikaelSmith) [inline man/man8/facter.8]: On Mac it does
```
OPTIONS
             --color                      Enable color output.
             --config                     A custom location for the config file.
             --custom-dir arg             A directory to use for custom facts.
       -d, [ --debug ]                    Enable debug output.
             --external-dir arg           A directory to use for external facts.
             --help                       Print help and usage information.
       -j, [ --json ]                     Output facts in JSON format.
             --list-block-groups          List the names of all blockable fact groups.
             --list-cache-groups          List the names of all cacheable fact groups.
             --show-legacy                Show legacy facts when querying all facts.
       -l, [ --log-level ] arg (=warn)    Set logging level.
                                          Supported levels are: none, trace, debug,
                                          info, warn, error, and fatal.
             --no-block                   Disable fact blocking.
             --no-cache                   Disable fact caching.
             --no-color                   Disable color output.
             --no-custom-fact             Disable custom facts.
             --no-external-facts          Disable external facts.
             --no-ruby                    Disable loading Ruby, facts requiring Ruby, and custom fact
s.
             --trace                      Enable backtraces for custom facts.
             --verbose                    Enable verbose (info) output.
       -v, [ --version ]                  Print the version and exit.
       -y, [ --yaml ]                     Output facts in YAML format.
```

Comment (Magisus) [inline man/man8/facter.8]: Yes, I had tried it at least on Mac, and actually fixed some spacing that was bad before.

**Pull request merged**